conda_smithy/templates/circle.yml.tmpl

```diff
@@ -24,14 +24,31 @@ dependencies:
   override:
     - docker pull condaforge/linux-anvil
 
+{%- if matrix %}
+{%- if 'ccache-toolchain' in ' '.join(package['meta']['requirements']['build']) %}
```

Comment:
Just thinking about what would be done if we decide to use ccache by default.
We would move ```ccache-toolchain``` code to ```toolchain```. Then this ```if``` would check for ```toolchain``` package instead, do you agree?

Also we could use ```conda-forge.yml``` for that. It would be more explicit and make it possible to disable ccache stuff altogether.